### DISCUSSION TITLE
Taxonomies and docs-only site not working

### DISCUSSION THREAD

**Discussion by honzik20** [google/docsy#698] [Q&A] [answered]
The mostly docs concept is great and I've adopted it for my own site but I can't seem to make taxonomies work with it. 

If my content is under en/, the terms pages don't list any topics - it seems they're not picking up the correct template (something to do with cascade?).

If I keep docs under en/docs, it works - but that means I need a homepage. 

I've tried enabling taxonomies on https://github.com/gwatts/mostlydocs repo and I don't even see a terms page.

Does anyone have any advice?

**Comment by narrenfrei** [comment]
Have you updated to the current Docsy version?
https://github.com/gwatts/mostlydocs/tree/main/themes has a 6 months old Docsy.

**Reply by honzik20** [reply]
Thanks @narrenfrei, I've updated it to the latest - and now at least the taxonomy terms pages appear. 

But it's got the same problem as my repo - none of the topics with that tag is being listed.

**Reply by narrenfrei** [reply]
Have you already read the Docsy Docs for taxonomy support?
https://www.docsy.dev/docs/adding-content/taxonomy/

Is it possible to get a link to your repo and/or webpage? Or a minimal example if it's not public?

**Reply by honzik20** [reply]
Thanks for the help @narrenfrei.

I have read the docs. I've got it working, but only when content is in a docs/ folder.

I've created a repo based on mostlyDocs with taxonomies enabled (and submodules updated): https://github.com/honzik20/mostlyDocs

Note: Moved comment because I hadn't replied to the thread.

**Reply by narrenfrei** [reply]
Yesterday evening I found time to take a look on your issue. But until now I haven't fount a solution. It seems to me as would Docsy or Hugo make a other use of the templates (or at least are the variables with the page lists structured in another way), when you define something like:

```
cascade:
- type: "blog"
  # set to false to include a blog section in the section nav along with docs
  toc_root: true
  _target:
    path: "/blog/**"
- type: "docs"
  _target:
    path: "/**"
```

Maybe @gwatts  or @LisaFC  know a little bit more about the cascade options?
I will try to find out more in the next days.

**Reply by LisaFC** [reply]
The mostly docs example is @gwatts so he may know more about how the cascading works, but I'll try to take a look as well. Sometimes Hugo is ... less than intuitive.

**Comment by honzik20** [comment, marked as answer]
I've created an issue from this discussion because it doesn't seem there's a configuration fix: https://github.com/google/docsy/issues/716

**Reply by gwatts** [reply]
Thanks for all the feedback; per discussion on the linked issue #716 new updates have been pushed to the mostlydocs repo and taxonomies are turned on by default there now 😄
